code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -14,7 +14,7 @@
 	//  = - Strict type matching.  Will NOT check for subtypes.
 	var/list/can_hold = new/list() //List of objects which this item can store (if set, it can't store anything else)
 	var/list/cant_hold = new/list() //List of objects which this item can't store (in effect only if can_hold isn't set)
-	var/list/ignore_w_class = new/list() //List of objects which will fit in this item, regardless of size. AKA can_hold_too.
+	var/list/ignore_w_class = new/list() //List of objects which will fit in this item, regardless of size. Doesn't restrict to ONLY items of these types. (in effect only if can_hold isn't set)
```

Comment:
can_hold = can_only_hold (for belts n shiet)
max_w_class = can_hold_everything_but_only_up_to_this_size (general "classic" containers like boxes and backpacks)
ignore_w_class = can_hold_anyways_regardless_of_can_hold_everything_but_only_up_to_this_size